main.py

```diff
@@ -33,15 +33,61 @@ def index():
     }
     return jsonify({"documents": payload}), 200
 
-@app.route("/spots")
+
+@app.route("/api/spots", methods=["GET", "POST"])
 def spots():
-    # Get list of all spots in the database
-    spots = list(Spot.objects.all().values())
-    return Response(
-        response=JSONEncoder().encode(spots),
-        status=200,
-        mimetype="application/json"
-    )
+    if request.method == "POST":
+        # Post a spot to the database
+        body = request.json
+        if not "name" in body or not "description" in body:
```

Comment:
We'd ideally have some sort of JSON response with the status code / error message as part of the response body.

Reply:
Could you clarify on this? We want to have the status code in the response and in the the response body?

Reply:
It feels icky to me for our API to return non-JSON formatted data. We don't necessarily have to include the status code, but having something like:
```
{
    "success": false,
    "message": "Missing parameters"
}
```
seems a bit nicer, no?

Reply:
I think the success parameter is unnecessary due to the status code implying success or failure, but I agree that formatting the message like this is a good idea, will change.